### ISSUE TITLE
Original link: is it a link or a button?

### ISSUE THREAD

**Issue by afercia** [voldemortensen/menu-customizer#108]
When using an `a` element then there's no need for a keydown event, unless I'm missing something. Just a click and preventDefault().

**Comment by celloexpressions** [comment]
It's a link that needs to open the URL in the preview instead of navigating to it. So you think we can just remove the keydown event and this'll be resolved?

**Comment by afercia** [comment]
Agreed it's a link (it also allows you to right click and copy the URL) so semantically it's more a link than a button. Yup, I see no reasons for the keydown event, unless I'm missing something.

**Comment by ocean90** [comment]
Keydown event was removed in 6accb7a78d557de804f72f4559cdaa16afd715d4
